src/lasp_advertisement_counter_client.erl

```diff
@@ -100,15 +100,16 @@ handle_info(log, #state{impressions=Impressions}=State) ->
     schedule_logging(),
 
     {noreply, State};
-handle_info(view, #state{actor=Actor, impressions=Impressions}=State0) ->
+
+handle_info(view, #state{actor=Actor, impressions=Impressions0}=State) ->
```

Comment:
`impression` was already there.
I'm just stopping the schedule if it reaches max.